src/services/coreData/base.ts

```diff
@@ -169,6 +169,24 @@ class Base {
     return { items };
   }
 
+  /**
+   * Returns the related media contents for the record with the passed ID.
+   *
+   * @param id
+   */
+    async getRelatedMedia(id: string) {
```

Comment:
We'll also need to update the content loaders to fetch related media contents for this to work in static mode.